generate.yaml

```diff
@@ -0,0 +1,567 @@
+asyncapi: '2.0.0'
```

Comment:
Was this file supposed to be committed? It has no references.

Reply:
I used it for testing and forgot about it. Removed, thanks!